src/autorest-core/resources/default-configuration.md

```diff
@@ -114,11 +114,11 @@ pipeline:
   swagger-document/azure-openapi-validator:
     input:
       - transform
-      - azure-validator # artificial predecessor in order to ensure order of messages for CI purposes
+      - azure-validator 
```

Comment:
With https://github.com/Azure/autorest/pull/2326#discussion_r120515648 I meant that the comment was simply ignored and moved to a different place, sort of as if you remove a "please keep door closed" from a door, open the door, and put the sign on a wall.